client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
+
+.PinkHeader {
+    background-image: var(--profile-banner);
+    background-repeat:no-repeat;
+    background-size: cover;
+    width: 1440px;
+    height: 231px;
+    text-align: center;
+    padding-top: 3px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+h3{
+    font-family: 'Reem Kufi', sans-serif;
+    color: #FAFAFA;
+    font-size: 48px;
+    line-height: 72px;
+    text-align: center;
+    margin: 25px 0 0 0;
+}
+
+.box2{   
+    margin: 0 auto;
+    width: 356px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+}
+
+
+.container-1 input#search {
+    width: 356px;
+    height: 48px;
+    background-color: #FAFAFA;
+    border: 1px solid #E5E5E5;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 36px;
+    padding: 15px 0 10px 0;
+    font-family: 'Reem Kufi', sans-serif;
+    color:  rgba(37, 37, 37, 0.3);
+    padding-left: 65px;
+    border-radius: 3px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.container-1 .icon{
+    position: absolute;
+    top: 47%;
+    margin-left: 25px;
+    margin-top: 14px;
+    z-index: 1;
+}
+
+img{
```

Comment:
Can you provide a class name here? Otherwise these stills will clash with other image styling in the app

Reply:
done